airflow/www_rbac/static/webpack.config.js

```diff
@@ -0,0 +1,128 @@
+/**
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements. See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership. The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License. You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing,
+* software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+* KIND, either express or implied. See the License for the
+* specific language governing permissions and limitations
+* under the License.
+*/
+
+const webpack = require('webpack');
+const path = require('path');
+const ManifestPlugin = require('webpack-manifest-plugin');
+const CleanWebpackPlugin = require('clean-webpack-plugin');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
> Since webpack v4 the extract-text-webpack-plugin should not be used for css. Use mini-css-extract-plugin instead.

What does this plugin do anyways?

Reply:
nice catch about the deprecation!!! 
So, if you have some CSS code (hard-coded or imported) in a JS file, this plugin will extract that in a separate `.css or .min.css` file and then we can use that .min.css file in our code.

Reply:
(... how can you have CSS code in a JS file?)

Reply:
https://github.com/webpack-contrib/mini-css-extract-plugin/tree/master/test/cases
P.S: I'm also using this plugin to combine multiple css files together in one minified css.